ci/jobs/build_clickhouse.py

```diff
@@ -104,6 +100,13 @@ def main():
 
     cmake_cmd = BUILD_TYPE_TO_CMAKE[build_type]
     info = Info()
+    if not info.is_local_run:
+        os.environ["SCCACHE_BUCKET"] = Settings.S3_ARTIFACT_PATH
+        os.environ["SCCACHE_S3_KEY_PREFIX"] = "ccache/sccache"
+        os.environ["CTCACHE_DIR"] = "ccache/clang-tidy-cache"
```

Comment:
I think setting `CTCACHE_DIR` (used for setting a local cache) is unneeded if we're always going to use an S3 bucket